### ISSUE TITLE
 error TS2307: Cannot find module 'ngx-fullcalendar'.

### ISSUE THREAD

**Issue by lilworks** [StickNitro/ngx-fullcalendar#5]
I tried the solution gived in issue 2  adding --save to ngx-fullcalendar install. But I still get:
 error TS2307: Cannot find module 'ngx-fullcalendar'.

Here my angular depedencies:
"dependencies": {
    "@angular/animations": "^6.1.0",
    "@angular/common": "^6.1.0",
    "@angular/compiler": "^6.1.0",
    "@angular/core": "^6.1.0",
    "@angular/forms": "^6.1.0",
    "@angular/http": "^6.1.0",
    "@angular/platform-browser": "^6.1.0",
    "@angular/platform-browser-dynamic": "^6.1.0",
    "@angular/router": "^6.1.0",
    "core-js": "^2.5.4",
    "dragula": "^3.7.2",
    "fullcalendar": "^4.0.0-alpha",
    "fullcalendar-scheduler": "^4.0.0-alpha.2",
    "moment": "^2.22.2",
    "ngx-fullcalendar": "^4.0.4",
    "rxjs": "~6.2.0",
    "zone.js": "~0.8.26"
  },

**Comment by StickNitro** [comment]
Able to replicate using `yarn install` with `v4.0.4`, rolling back to `v4.0.3` using `yarn` seems to work fine so suggest using 4.0.3 as a workaround until fixed

Interestingly, v4.0.4 works fine when using NPM

**Comment by JStw** [comment]
I tried with npm and I've got the same issue only for 4.0.4. In 4.0.3, it found the module.

**Comment by StickNitro** [comment]
I believe this was caused when I moved the project out of the application where it was created but did not move the project out correctly, consequently the project structure was not correct causing this issue.

I have updated the project structure for a library, this corrected the project format and I have added a demo (ngx-fullcalendar-test) application alongside the library. This is published as v4.0.5.

Closing this now as fixed, open a new issue for any further problems
